pkgs/by-name/co/cosmic-wallpapers/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "cosmic-wallpapers";
+  version = "0-unstable-2024-09-27";
```

```suggestion
  version = "1.0.0-alpha.2";
```

Please use the upstream tags. See other cosmic-* packages for more information.